webcompat/form.py

```diff
@@ -445,7 +447,7 @@ def domain_name(url):
     return domain
 
 
-def build_formdata(form_object):
+def build_formdata(form_object, extra_metadata=None):
```

Comment:
Probably it would be better to have something like `add_metadata` to modify the `form` dictionary. Specifically when `build_formdata` has already mechanism for adding metadata into the form.